.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
You usually just update the root .gitignore to ignore the whole .idea folder.

If there are specific files you do want, then the root gitignore can also specify that. In general you don't want to include much if anything from IDE folders